packages/@tinacms/graphql/src/database/index.ts

```diff
@@ -338,6 +338,7 @@ export class Database {
     const graphqlPath = path.join(GENERATED_FOLDER, `_graphql.json`)
     return this.store.get(normalizePath(graphqlPath))
   }
+  //TODO - is there a reason why the database fetches some config with "bridge.get", and some with "store.get"?
```

Comment:
cc @jeffsee55 , 

> is there a reason why the database fetches some config with "bridge.get", and some with "store.get"? Wondering if it could make sense to split some of these functions into multiple classes.

Reply:
There's a lot of nuance here because we needed to continue to support multiple stores/bridges at the same time. In general I expect these differences to go away when we remove the logic for non-indexable stores (ie. `Filesystem`). I imagine that will need to be a large-ish refactor and it'd be best to leave this alone for now.